public/sketch.js

```diff
@@ -34,3 +35,29 @@ function mouseDataSend() {
 		socket.emit('mouse', data);
 	}
 }
+
+// If you press 'p' on your keyboard this function will save the canvas
+// in its current state to a jpg in your downloads.
+// This is what we'd want to send to the database.
+function keyPressed(event) {
```

Comment:
this is crazy! all native p5.js stuff??

Reply:
@jffhtchr Yep! More than a few times I've custom made an awkward function only to find that p5 has a built-in function, haha.